app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt

```diff
@@ -41,11 +29,7 @@ import org.mozilla.fenix.R
 import org.mozilla.fenix.components.FenixSnackbar
 import org.mozilla.fenix.components.StoreProvider
 import org.mozilla.fenix.components.metrics.Event
-import org.mozilla.fenix.ext.bookmarkStorage
-import org.mozilla.fenix.ext.components
-import org.mozilla.fenix.ext.minus
-import org.mozilla.fenix.ext.nav
-import org.mozilla.fenix.ext.toShortUrl
+import org.mozilla.fenix.ext.*
```

Comment:
Is this bad?

Reply:
Allow me to add my two cents: Most vote against using wildcard imports; it tends to increase coupling, especially outside the IDE. 

Also this is against ktlint & detekt rules. You can find more in this [thread](https://github.com/android/kotlin-guides/issues/65).

Reply:
Right, our lint rules don't let you do this. You may need to change your Kotlin preferences in Android Studio so that it doesn't automatically star import for anything except synthetic views :/ Can help you do that if you want